src/sequelize/models/User.js

```diff
@@ -33,6 +33,11 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       defaultValue: 'user',
     },
+    verified: {
```

Comment:
the datatype should be `DataTypes.BOOLEAN`. We expect a user to either be verified or not. A True vs False value

Reply:
I'm gonna implement that.